src/Groups/GroupDetails/GroupDetailsContainer.js

```diff
@@ -198,6 +198,10 @@ export const GroupDetailsContainer = ({
       .finally(() => setIsLoading(false));
   };
 
+  const onBatchAllocationLogs = useCallback(() => {
+    history.push({ pathname: `${GROUPS_ROUTE}/${groupId}/batch-allocations/logs` });
```

Comment:
You are losing the `search`, when you get back from that page the filters will be lost.

Example https://github.com/folio-org/ui-finance/blob/e005707b70a5df45f42d766175927b2cc921d1ff/src/Ledger/RolloverLogs/RolloverLogs.js#L74